src/loader/loader.c

```diff
@@ -500,7 +507,11 @@ loader_process_utility_hook(PlannedStmt *pstmt, const char *query_string,
 	else
 		process_utility = standard_ProcessUtility;
 
+#if PG13_GE
+	process_utility(pstmt, query_string, context, params, queryEnv, dest, qc);
+#else
 	process_utility(pstmt, query_string, context, params, queryEnv, dest, completion_tag);
+#endif
```

Comment:
If you use the same parameter name in the change above, you do not need this change.